decisions/0004-issue-triage.md

```diff
@@ -10,7 +10,7 @@ Approved
 
 The NIST OSCAL Team must plan work to add, change, or remove code in project repositories. We document these plans in GitHub issues, and implement the changes as specified in these issues with pull requests. Some of these issues are decided internally by the NIST OSCAL Team, others are requested by external community members. In the past and present, many issues, especially requests from external community members, are created and do not receive timely feedback. Initially or over time, it is not clear when NIST staff have officially received a request, intend to accept the work recommendation or reject it, and an approximate time window for when that work should start.
 
-In response to this, the former technical director and a subset of the team were involved in issue triage informally. Those team members [implemented some automation to process incoming requests](https://github.com/usnistgov/OSCAL/blob/60918cd3fc5508b69b71e8691b071108dab46ec9/.github/workflows/issue-triage.yml) and assist with ongoing triage by adding them to [a specific GitHub project, distinct from the (now legacy) project that tracked each sprint, for managing triage in bi-weekly meetings](https://github.com/usnistgov/OSCAL/projects/52). Subsequently, the team migrated sprint and overall backlog tracking from one version of project boards in GitHub, both the Issue Triage board and other uses in OSCAL-related projects, to [a newer cross-repository project board with other usability enhancements](https://github.com/orgs/usnistgov/projects/25/). The consequence of this change was the issue triage board, and process more generally, became more and more disjointed from the more recent process and tooling for planning work through issue management.
+In response to this, the former technical director and a subset of the team were involved in issue triage informally. Those team members [implemented some automation to process incoming requests](https://github.com/usnistgov/OSCAL/blob/60918cd3fc5508b69b71e8691b071108dab46ec9/.github/workflows/issue-triage.yml) and assist with ongoing triage by adding them to ["Needs Triage" in the NIST OSCAL Work Board](https://github.com/orgs/usnistgov/projects/25). Subsequently, the team migrated sprint and overall backlog tracking from one version of project boards in GitHub, both the Issue Triage board and other uses in OSCAL-related projects, to [This newer cross-repository project board with other usability enhancements](https://github.com/orgs/usnistgov/projects/25/). The consequence of this change was the issue triage board, and process more generally, became more and more disjointed from the more recent process and tooling for planning work through issue management.
```

Comment:
Non-blocking: why reformatting?

Reply:
this is because GSA fork is not synced with develop